added delete comment functionallity and modified post comments

OzRikT95 opened the pull request.

nc-ali-abdallah reviewed (commented):
If I post a new comment and then press the delete button straight away the comment shows as deleted.
If I then refresh the page the comment reappears, are you only deleting it in state when the comment is first optimistically shown?

OzRikT95 commented:
I’m unsure of this, as I only set the comment to optimistically show after I added the delete button functionality.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
From: nc-ali-abdallah ***@***.***>
Sent: Thursday, March 14, 2024 11:56:13 AM
To: OzRikT95/fe-nc-news ***@***.***>
Cc: Rikki Day ***@***.***>; State change ***@***.***>
Subject: Re: [OzRikT95/fe-nc-news] added delete comment functionallity and modified post comments (PR #6)


@nc-ali-abdallah commented on this pull request.

If I post a new comment and then press the delete button straight away the comment shows as deleted.
If I then refresh the page the comment reappears, are you only deleting it in state when the comment is first optimistically shown?

—
Reply to this email directly, view it on GitHub<https://github.com/OzRikT95/fe-nc-news/pull/6#pullrequestreview-1936452860>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A3ILRKXM3IHHNKYBQLO7B63YYGF53AVCNFSM6AAAAABEV5RNJKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTSMZWGQ2TEOBWGA>.
You are receiving this because you modified the open/close state.Message ID: ***@***.***>
